src/main/java/com/epam/helpers/ErrorMessages.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.helpers;
+
+public enum ErrorMessages {
```

Comment:
I think, it will be good write error messages in the .property file, but i won't be against to the Enum

Reply:
okay